scripts/crosstool-NG.sh

```diff
@@ -733,11 +733,15 @@ if [ "${CT_LOG_TO_FILE}" = "y" ]; then
     fi
 fi
 if [ "${CT_PREFIX_DIR_RO}" = "y" ]; then
-    chmod -R a-w "${CT_PREFIX_DIR}"
+    # Since GNU coreutils 9.0, trying to change the mode of any symbolic link will cause chmod to exit with status 1.
+    # https://github.com/coreutils/coreutils/commit/bbb19b18687d5572dcbe7bbb1a48c8ee0f21786b
```

Comment:
I don't think this comment adds anything. If anyone wants to know the reason for this change they can look at the commit message.